subsys/bluetooth/controller/hci_internal.c

```diff
@@ -1200,6 +1204,7 @@ static uint8_t vs_cmd_put(uint8_t const * const cmd,
 #endif
 #if defined(CONFIG_BT_CTLR_LE_POWER_CONTROL)
 	case SDC_HCI_OPCODE_CMD_VS_WRITE_REMOTE_TX_POWER:
+	case SDC_HCI_OPCODE_CMD_VS_SET_AUTO_POWER_CONTROL_REQUEST_PARAM:
```

Comment:
Is it intentional that there is a fallthrough here?

Reply:
No, will fix. Thanks!